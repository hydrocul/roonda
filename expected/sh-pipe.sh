echo 'Hey!' | cat | perl $ROONDA_TMP_PATH/roonda_3772d206f212590b290e2779a784416fef72956e.pl
echo abc > $ROONDA_TMP_PATH/abc.txt
cat $ROONDA_TMP_PATH/abc.txt | cat
cat $ROONDA_TMP_PATH/abc.txt
cat $ROONDA_TMP_PATH/abc.txt| tee $ROONDA_TMP_PATH/abc1.txt > $ROONDA_TMP_PATH/abc2.txt
cat $ROONDA_TMP_PATH/abc1.txt
cat $ROONDA_TMP_PATH/abc2.txt

#################################################
# roonda_3772d206f212590b290e2779a784416fef72956e.pl:
#################################################
# print "Hello\n";
