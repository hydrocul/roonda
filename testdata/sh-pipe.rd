(sh v1)

(pipe (echo "Hey!")
      (cat)
      ((perl) (print "Hello\n")))

(pipe (echo "abc")
      (> (strcat (ref ROONDA_TMP_PATH) "/" abc.txt)))

(pipe (< (strcat (ref ROONDA_TMP_PATH) "/" abc.txt))
      (cat))

(pipe (< (strcat (ref ROONDA_TMP_PATH) "/" abc.txt)))

(pipe (< (strcat (ref ROONDA_TMP_PATH) "/" abc.txt))
      (> (strcat (ref ROONDA_TMP_PATH) "/" abc1.txt) (strcat (ref ROONDA_TMP_PATH) "/" abc2.txt)))

(pipe (< (strcat (ref ROONDA_TMP_PATH) "/" abc1.txt)))

(pipe (echo "ABC")
      (>> (strcat (ref ROONDA_TMP_PATH) "/" abc2.txt)))

(pipe (< (strcat (ref ROONDA_TMP_PATH) "/" abc2.txt)))

