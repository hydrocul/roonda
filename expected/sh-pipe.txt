Hello
abc
abc
abc
abc
